DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/os/recoverable_sysctl.py

```diff
@@ -0,0 +1,31 @@
+from dent_os_testbed.lib.os.sysctl import Sysctl
+
+
+class RecoverableSysctl(Sysctl):
+
+    saved_values = dict()
+
+    async def set(*argv, **kwarg):
+        devices = kwarg['input_data']
+        kwarg['parse_output'] = True
```

Comment:
What if the user passes `-n` or `-v`?

Reply:
Then we will have an error in parse_get function which is a part of sysctl API. Will add a fix in this PR.